web/js/components/timeline/timeline-data/info-modal.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+const DataPanelInfoModal = () => (
+  <div className="data-panel-info">
+    <header>
+      <p>
+        Temporal coverage of activated time-varying layers are shown in the layer coverage panel as striped blue lines in relation to the current timeline zoom level. The total temporal coverage dates are shown on the right.
+      </p>
+      <p>
+        Time-varying layers that are temporally coincident/have matching temporal coverage are shown in the timeline as a solid blue line, making it easier to find out when multiple layers have concurrent imagery available.
+      </p>
+    </header>
+    <div className="data-panel-info-item-container">
+      <section className="data-panel-info-item-left">
+        <img src="images/dp-axis.png" />
+      </section>
+      <section className="data-panel-info-item-right">
+        <div>
+          <p>
+            The “Include Hidden Layers” toggle will display hidden layers in the layer coverage panel and may affect the display of the matching coverage solid blue line.
```

Comment:
I think what this toggle does is pretty well explained by the label so maybe this can be condensed to something like:

"Including layers that are hidden in the layer coverage panel will take their dates into account when calculating concurrent coverage and, may affect the display of the timeline's solid blue overlapping coverage line."